src/components/ValidationProcess/ValidationProcess.jsx

```diff
@@ -1,38 +1,17 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { CheckCircle } from '@edx/paragon/icons';
 import {
   Form, Icon, PageBanner, Stack,
 } from '@edx/paragon';
-import { CheckCircle } from '@edx/paragon/icons';
-import { useState } from 'react';
-import { getLastReviewEventInfo, getSubmissionInfo } from '../../utils/helpers';
-import { CourseSubmissionInfo } from './CourseSubmissionInfo';
-import { ValidatorReview } from './ValidatorReview';
 
-// TODO: How will be received the current reviewer?
-const courseSelected = {
-  course_name: 'Test Course #1',
-  course_id: 'course-v1:org1+test-course-1',
-  organization: 'Organization 1',
-  categories: ['Category 1', 'Category 2'],
-  validation_body: 'Validator Body #1',
-  validation_process_events: [
-    {
-      status: 'Exempt',
-      created_at: '2023-08-01',
-      reason: null,
-      comment: 'This is the comment of the course author. This is the comment of the course author. This is the comment of the course author',
-      user: 'Course Author 1',
-    },
-    {
-      status: 'Disapproved',
-      created_at: '2023-08-15',
-      reason: 'Reason 1',
-      comment: 'This is the comment of Validator 1 This is the comment of Validator 1 This is the comment of Validator 1 This is the comment of Validator 1',
-      user: 'Validator 1',
-    },
-  ],
-};
+import { ValidatorReview } from './ValidatorReview';
+import { CourseSubmissionInfo } from './CourseSubmissionInfo';
+import { getLastReviewEventInfo, getSubmissionInfo } from '../../utils/helpers';
 
 const ValidationProcess = () => {
+  const courseSelected = useSelector((state) => state.currentValidationRecord);
```

Comment:
I think this follows the same logic as Timeline component, use props to display the info, because you are accessing the same information twice here and the parent component.